apps/modernization-api/src/main/java/gov/cdc/nbs/patient/profile/create/PatientCreateController.java

```diff
@@ -145,7 +150,14 @@ public PatientIdentifier create(@RequestBody NewPatient newPatient) {
         identificationService.add(context, newPatientIdentificationInput);
       });
     }
-
+    if (newPatient.ethnicity() != null) {
+      EthnicityInput ethnicityInput;
```

Comment:
question:  This is converting `EthnicityDemographic` -> `EthnicityInput` -> `PatientCommand.UpdateEthnicityInfo` would it make sense to include this in [PatientCreationService](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-api/src/main/java/gov/cdc/nbs/patient/profile/create/PatientCreationService.java) we can skip one transformation and go from `EthnicityDemographic` -> `PatientCommand.UpdateEthnicityInfo`

Reply:
Yes.  Let me pull the latest and integrate with that...